app/src/main/java/org/systers/mentorship/models/Task.kt

```diff
@@ -18,7 +18,7 @@ import kotlinx.android.parcel.Parcelize
 data class Task(
         val id: Int,
         val description: String,
-        val isDone: Boolean,
         val createdAt: Float,
-        val completedAt: Float
+        val completedAt: Float?,
+        val isDone: Boolean = completedAt != null
```

Comment:
Why have add it to the constructor? You don't need it there you can just declare it as a separate class variable.

Reply:
oh k

Reply:
done